benchmarks/in-memory-analytics/README.md

```diff
@@ -36,63 +36,66 @@ More information about the dataset is available at
 
 ### Running the Benchmark
 
-The benchmark runs the ALS algorithm on Spark through the spark-submit script
-distributed with Spark. It takes two arguments: the dataset to use for
-training, and the personal ratings file to give recommendations for. Any
-remaining arguments are passed to spark-submit.
+The benchmark runs the ALS algorithm on Spark through the spark-submit
+script distributed with Spark. It takes two arguments: the dataset to
+use for training, and the personal ratings file to give
+recommendations for. Any remaining arguments are passed to
+spark-submit.
 
-The cloudsuite/movielens-dataset image has two datasets (one small and one
-large), and a sample personal ratings file.
+The cloudsuite/movielens-dataset image has two datasets (one small and
+one large), and a sample personal ratings file.
 
-To run a benchmark with the small dataset and the provided personal ratings
-file:
+To run a benchmark with the small dataset and the provided personal
+ratings file:
 
-    $ docker create --name data cloudsuite/movielens-dataset
-    $ docker run --rm --volumes-from data cloudsuite/in-memory-analytics \
-        /data/ml-latest-small /data/myratings.csv
+    $ docker create --name data cloudsuite/movielens-dataset $ docker
+    run --rm --volumes-from data cloudsuite/in-memory-analytics \
+    /data/ml-latest-small /data/myratings.csv
 
 ### Tweaking the Benchmark
 
-Any arguments after the two mandatory ones are passed to spark-submit and can
-be used to tweak execution. For example, to ensure that Spark has enough memory
-allocated to be able to execute the benchmark in-memory, supply it with
---driver-memory and --executor-memory arguments:
+Any arguments after the two mandatory ones are passed to spark-submit
+and can be used to tweak execution. For example, to ensure that Spark
+has enough memory allocated to be able to execute the benchmark
+in-memory, supply it with --driver-memory and --executor-memory
+arguments:
 
-    $ docker run --rm --volumes-from data cloudsuite/in-memory-analytics \
```

Comment:
Good job. :+1: 
Why did you change the `in-memory-analytics` README file?
Also, could you please give me these details:
Path to all dockerfiles with their repo names (on DockerHub), and their tags.